src/window/keyboard_manager.rs

```diff
@@ -95,7 +95,24 @@ impl KeyboardManager {
             KeyCode::NumpadSubtract => Some("kMinus"),
             KeyCode::NumpadAdd => Some("kPlus"),
             KeyCode::NumpadEnter => Some("kEnter"),
-            KeyCode::NumpadDecimal => Some("kDel"),
+            KeyCode::NumpadEqual => Some("kEqual"),
+            KeyCode::NumpadComma => match key_event.logical_key.as_ref() {
+                Key::Character(",") => Some("kComma"),
+                Key::Character(".") => Some("kPoint"),
+                _ => key_event.logical_key.to_text(),
```

Comment:
I think it's better to return None here, that way `format_normal_key` will be called and used, which probably works better.

And the same thing for `NumpadDecimal`

Reply:
Here you go